### PR TITLE
clean: Revert workaround from #146

### PR DESCRIPTION
**Pull request by prcr**
Reverts #146. The previous workaround should no longer be necessary, as CircleCI is already updating Homebrew on macOS images as a workaround on their side:

https://discuss.circleci.com/t/macos-image-users-homebrew-brownout-2021-04-26/39872/2

### CODE REVIEW

**Comment by prcr** [comment]
We can't merge this pull request just yet after all, since CircleCI removed the patch on their side :grimacing: 

https://discuss.circleci.com/t/macos-image-users-homebrew-brownout-2021-04-26/39872/3

**Comment by prcr** [comment]
According to the latest update, we can now revert the workaround:

https://discuss.circleci.com/t/macos-image-users-homebrew-brownout-2021-04-26/39872/10

**Review by rubencodacy — APPROVED** [review]
LGTM

**Review by rubencodacy — APPROVED** [review]

**Pull request merged**
